src/MSALWrapper/ExceptionListToStringConverter.cs

```diff
@@ -74,11 +74,19 @@ private static string SingleLineException(Exception ex)
         {
             if (ex != null)

```

Comment:
What if we flipped this and did
```csharp
if (ex == null) {
    return "null";
}
```
and _then_ put the remainder of the logic? The early return would place the `"null"` closer to the condition that causes it and you wouldn't need the `else` clause.